src/orbits/kepler.jl

```diff
@@ -1,110 +1,230 @@
+using AstroLib: kepler_solver, trueanom
+using KeywordDispatch
+using PhysicalConstants
+using Unitful
+using UnitfulAstro
 
-using AstroLib: kepler_solver
+const G = PhysicalConstants.CODATA2018.G
+const G_val = ustrip(u"cm^3/g/s^2", G)
```

Comment:
```suggestion
const G_cgs = ustrip(u"cm^3/g/s^2", G)
```
I think this will greatly avoid developer confusion in the future.

Reply:
Great point, added!